static/terraboard.js

```diff
@@ -312,6 +324,27 @@ app.controller("tbSearchCtrl", ['$scope', '$http', '$location', '$routeParams',
         $scope.attrKey = undefined;
         $scope.attrVal = undefined;
         $scope.results = undefined;
-        $scope.doSearch(1);
+        $location.url('/search');
```

Comment:
Two comments here:
* absolute URLs break with the base href, only use relative URLs;
* why not use `$location.search()` instead?